server/parsec/events.py

```diff
@@ -31,8 +31,8 @@
 # can be passed with the event (on top of that we keep the events in cache for SSE
 # last-event-id, so better avoid too much pressure of the RAM)
 # We keep a large margin with PostgreSQL limit given the event also contains additional
-# fields and must be encoded in base64 (which alone adds ~33% in size).
-EVENT_VLOB_MAX_BLOB_SIZE = 4096
+# fields and the blob is hex encoded.
+EVENT_VLOB_MAX_BLOB_SIZE = 1024
```

Comment:
Why 1024 and not 2048 ?

Reply:
Honestly, I took an extra margin so we don't run again into this kind of issue.

Adding a test would have been better, but as @mmmarcos mentioned it, we should simply get rid of the vlob data in the vlob event: https://github.com/Scille/parsec-cloud/issues/8420